AsyncDisplayKit/ASCellNode.h

```diff
@@ -107,6 +111,7 @@ typedef NSUInteger ASCellNodeAnimation;
  */
 - (instancetype)initWithViewControllerBlock:(ASDisplayNodeViewControllerBlock)viewControllerBlock didLoadBlock:(ASDisplayNodeDidLoadBlock)didLoadBlock;
 
+- (void)updateScrollSituationWithScrollVIew:(UIScrollView *)scrollView;
```

Comment:
Let's call this visibleNodeScrollViewDidScroll:(UIScrollView *)scrollView withCellFrameInScrollView:(CGRect)cellFrame